### DISCUSSION TITLE
OrderList dragdrop property not working in v18

### DISCUSSION THREAD

**Discussion by jaime-clavei** [primefaces/community#2909] [PrimeNG] [answered]
The drag and drop isn't working in the OrderList component. Not sure if it's a known issue. It can be tested in the [order list page](https://v18.primeng.org/orderlist)

**Comment by qwadrox** [comment, marked as answer]
@jaime-clavei I created an issue: https://github.com/primefaces/primeng/issues/16633

**Reply by jaime-clavei** [reply]
Thank you!
